packages/meditrak-server/src/dataAccessors/updateOrCreateSurveyResponse.js

```diff
@@ -68,12 +69,19 @@ export async function updateOrCreateSurveyResponse(models, surveyResponseObject)
     // Ensure data_time is populated, supporting legacy versions of MediTrak
     surveyResponseProperties.data_time = getDataTime(surveyResponseObject);
 
+    // If the response is for a survey where approval is required, set approval to pending
+    let approvalStatus = models.surveyResponse.approvalStatusTypes.NOT_REQUIRED;
+    if (survey.requires_approval) {
+      approvalStatus = models.surveyResponse.approvalStatusTypes.PENDING;
+    }
```

Comment:
Same change needed in `constructNewSurveyResponseDetails` within `importSurveyResponses`